extractor/src/main/java/org/schabi/newpipe/extractor/uih/UIHandler.java

```diff
@@ -0,0 +1,31 @@
+package org.schabi.newpipe.extractor.uih;
+
+import java.io.Serializable;
+
+public class UIHandler implements Serializable {
```

Comment:
Please don't abbreviate class names. Since this is a library, we should try to make names clear, even if they are long so they don't confuse the users. This is especially the case for `UIHandler`, I thought we are doing something graphical in the extractor when I saw this initially.

In case of `UrlIdHandler`, I think we should think of a different name for it, such as `LinkHandler`, because this now also contain other data. Similarly, we can simply drop the `Handler` suffix since now the class is just a pojo and doesn't *handle* any logic. 

This way, `UIHandler` class becomes `Link` class, `ListUIHandler` to `ListLink` and `SearchQIHandler` to `SearchQuery`. What do you think?

Reply:
I agree. I will rename them.